### DISCUSSION TITLE
Add a amount option when giving a player spawners

### DISCUSSION THREAD

**Discussion by DJDan21** [OmerBenGera/WildStacker#397] [Pending Feature Requests]
Hi,
I would like to make suggest a new variable to the current giving spawners to players command. I have provided you with an example of the command:
/stacker give <player> spawner <mobname> <stacksize> <amount> <upgrade>
I would you will consider my idea.

**Comment by OmerBenGera** [comment]
/ws give <player-name> spawner <type> <stack size>
If you dont want spawners to be given as one item, just disable it in your config file.

**Comment by DJDan21** [comment]
I want it as one item but I multiples of it as well, cause otherwise you have to spam the command to multiples stacks
